core/executor/src/precompiles/get_cell.rs

```diff
@@ -3,14 +3,15 @@ use evm::{Context, ExitError, ExitSucceed};
 
 use protocol::types::H160;
 
-use crate::err;
 use crate::precompiles::{precompile_address, PrecompileContract};
+use crate::system_contract::image_cell::CellKey;
+use crate::{err, system_contract::image_cell::ImageCellContract};
 
 #[derive(Default, Clone)]
 pub struct GetCell;
 
 impl PrecompileContract for GetCell {
-    const ADDRESS: H160 = precompile_address(0x04);
+    const ADDRESS: H160 = precompile_address(0xf0);
```

Comment:
Is there any special mean between `0x04` and `0xf0` ?

Reply:
Use `0xf0` rather than `0x04` aims at distinguish the precompile contracts included in EIP or not.